lib/kast.js

```diff
@@ -282,6 +283,8 @@ const format = (o, isRaw = false, mixFix = false, colorize = false) => {
       "intList" : "",
       ".List{\"intList\"}": "",
       ".WordStack_EVM-DATA": "",
+      "#WordPackUInt112UInt112UInt32(_,_,_)_RULES": "#WordPackUInt112UInt112UInt32",
```

Comment:
not sure we should be putting application specific rules into `klab` tbh

Reply:
I can agree about this one.

Reply:
I think I prefer to see the K version in this case. I think we should be careful not to hide meaning.